class Solution {
    public String reverseWords(String s) {
        String words[]=s.split("\\s+");
        String result="";
        for(int i=words.length-1;i>=0;i--){
            result+=words[i];
            result+=" ";
        }
        return result.trim();
    }
   
}